examples/python/signed_distance_fields/signed_distance_fields/__main__.py

```diff
@@ -117,7 +117,14 @@ def log_sampled_sdf(points: npt.NDArray[np.float32], sdf: npt.NDArray[np.float32
 
 def log_volumetric_sdf(voxvol: npt.NDArray[np.float32]) -> None:
     names = ["width", "height", "depth"]
-    rr.log("tensor", rr.Tensor(voxvol, dim_names=names))
+    # Use a symmetric value range, so that the `cyantoyellow` colormap centers around zero.
+    # Either positive or negative range might be quite small, so don't exceed 1.5x of either.
+    min_val = cast(float, np.min(voxvol))
+    max_val = cast(float, np.max(voxvol))
+    negative_range = max(0.0, abs(min_val))
+    positive_range = max(0.0, abs(max_val))
+    range = max(min(positive_range, negative_range * 1.5), min(negative_range, positive_range * 1.5))
```

Comment:
Is this the same? O.o My brain has a hard time following your code, but I suspect the same will be said by you of my code 😆 

```suggestion
	range = min(max(negative_range, positive_range), 1.5 * min(negative_range, positive_range))
```

Reply:
I think so, yeah. Liking yours better, captures the intention better!